soh/src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.c

```diff
@@ -35,7 +35,7 @@ void BossDodongo_DrawEffects(PlayState* play);
 void BossDodongo_UpdateEffects(PlayState* play);
 
 const ActorInit Boss_Dodongo_InitVars = {
-    ACTOR_EN_DODONGO,
+    ACTOR_BOSS_DODONGO,
     ACTORCAT_BOSS,
```

Comment:
This was already discussed in https://github.com/HarbourMasters/Shipwright/pull/1729 which is still open, but was actively hurting me here because it was also speeding up the regular dodongo's, so I decided to change it in this PR too.